front/src/modules/rollingStock/components/RollingStockEditor/formatSpreadSheetCurve.ts

```diff
@@ -1,22 +1,27 @@
 import type { DataSheetCurve, EffortCurveForm } from 'modules/rollingStock/types';
 import { kmhToMs } from 'utils/physics';
 
-const formatToEffortCurve = (rows: DataSheetCurve[]) =>
+const formatCurve = (rows: DataSheetCurve[]) =>
   rows.reduce<EffortCurveForm>(
     (result, row) => {
-      result.speeds.push(row.speed !== '' ? kmhToMs(Number(row.speed)) : undefined);
-      // Back-end needs effort in newton
-      result.max_efforts.push(row.effort !== '' ? Number(row.effort) * 1000 : undefined);
+      // Test whether the speed cell is null before converting
+      if (row.speed !== null) {
+        result.speeds.push(kmhToMs(Number(row.speed)));
+      } else {
+        result.speeds.push(null);
+      }
+
+      // Test whether the effort cell is null before converting
+      if (row.effort !== null) {
+        // Back-end needs effort in newton
+        result.max_efforts.push(Number(row.effort) * 1000);
+      } else {
+        result.max_efforts.push(null);
+      }
```

Comment:
I think this is enough :)
```suggestion
      result.speeds.push(row.speed !== null ? kmhToMs(Number(row.speed)) : null);
      // Back-end needs effort in newton
      result.max_efforts.push(row.effort !== null ? Number(row.effort) * 1000 : null);
```

Reply:
Ty, fixed :)